### PR TITLE
Fixing issues with the PR

### PR DESCRIPTION
**Pull request by lakinwecker**
I also snuck in the ability to run some of the make team stuff in parallel, which should make it much faster.

### CODE REVIEW

File: heltour/tournament/admin.py [hunk 1]
```diff
@@ -823,11 +812,15 @@ def insert_alternates(alts_split):
                                              board_number=board_number)
 
         season = get_object_or_404(Season, pk=object_id)
+        season_started = bool(Round.objects.filter(season=season, publish_pairings=True).count())
+        team_count = Team.objects.filter(season=season).count()
         if request.method == 'POST':
-            form = forms.CreateTeamsForm(request.POST)
+            if season_started:
+                return HttpResponse(status=400)
```
Comment (colwem) [inline heltour/tournament/admin.py]: Shouldn't we just disable the entire view if season_started?

File: heltour/tournament/forms.py [hunk 2]
```diff
@@ -368,4 +368,11 @@ class CreateTeamsForm(forms.Form):
...
                                help_text="Ratio of team members to alternates.  A value of 0.8 "
                                          "means 20% will be made alternates")
+    confirm_create = forms.BooleanField()
 
+
+
+    def __init__(self, team_count, *args, **kwargs):
+        super(CreateTeamsForm, self).__init__(*args, **kwargs)
+
+        self.fields['confirm_create'].label = f'Yes, I\'m sure - {team_count} teams and and regenerate'
```
Comment (colwem) [inline heltour/tournament/forms.py]: Is the english here correct?  Should be `f"Yes, I'm sure.  Delete {team_count} teams and regenerate"`

File: heltour/tournament/teamgen.py [hunk 3]
```diff
@@ -383,4 +386,21 @@ def reduce_variance(teams):
...
+    league['teams'] = reduce_variance(league['teams'])
+
+def get_best_league(player_data, boards, balance, count):
+    pool = Pool(getattr(settings, 'TEAMGEN_PROCESSES_NUMBER', 1))
+    args = [(player_data, boards, balance) for _ in range(count)]
+    leagues = pool.map(make_league_map, args)
+    max_happiness = max([total_happiness(l['teams']) for l in leagues])
+    happy_leagues = [l for l in leagues if total_happiness(l['teams']) == max_happiness]
+
+    pool.map(reduce_variance_map, happy_leagues)
```
Comment (colwem) [inline heltour/tournament/teamgen.py:401]: Very cool, thank you for showing me this 😄

File: heltour/tournament/admin.py (hunk 1 above)
Comment (lakinwecker) [reply heltour/tournament/admin.py]: You mean, move this to the top?

File: bin/createteams.py [hunk 4]
```diff
@@ -1,14 +1,17 @@
-# NOTE: click is not a dependency of this project, you'll have to manually
-#       install it to run this file.
-import click
+import sys
+try:
+    import click
+except ImportError:
+    sys.exit("""You have to manually click to run this file.""")
```
Comment (cyanfish) [inline bin/createteams.py]: "click" -> "install the 'click' package"

File: heltour/tournament/admin.py [hunk 5]
```diff
@@ -823,11 +812,15 @@ def insert_alternates(alts_split):
                                              board_number=board_number)
 
         season = get_object_or_404(Season, pk=object_id)
+        season_started = bool(Round.objects.filter(season=season, publish_pairings=True).count())
```
Comment (cyanfish) [inline heltour/tournament/admin.py]: Here (and below) you can use `.exists()` instead of `bool(count)`

File: heltour/tournament/forms.py (hunk 2 above)
Comment (lakinwecker) [reply heltour/tournament/forms.py]: will fix.

File: bin/createteams.py (hunk 4 above)
Comment (colwem) [reply bin/createteams.py]: Why the triple quotes?

Comment (lakinwecker) [reply bin/createteams.py]: No good reason. I originally had it as a multiple line string. Both things  have been addressed

**Pull request merged**